src/DataType/UserRequests.ts

```diff
@@ -14,36 +14,39 @@ export type UserRequest = {
 };
 
 export class UserRequests {
-    localforage_instance?: typeof localforage;
+    __localforage_instance?: typeof localforage;
```

Comment:
JS does have proper private fields: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes/Private_class_fields

Reply:
Forgot about those! Damn you, python.